opencga-app/app/scripts/azure/arm/appgw/azuredeploy.json

```diff
@@ -0,0 +1,356 @@
+        "apiVersion": "2018-08-01",
+        "location": "[resourceGroup().location]",
+        "sku": {
+            "name": "Basic"
+        },
+        "properties": {
+            "publicIPAllocationMethod": "Dynamic",
+            "dnsSettings":
+            {
+                "domainNameLabel": "[parameters('dnsNamePrefix')]"
+            }
+        },
+         "resources": [
+                {
+                    "type": "providers/diagnosticSettings",
+                    "name": "[concat('Microsoft.Insights/', variables('applicationGatewayPublicIpName'))]",
+                    "dependsOn": [
+                        "[resourceId('Microsoft.Network/publicIPAddresses',variables('applicationGatewayPublicIpName'))]"
+                    ],
+                    "apiVersion": "2017-05-01-preview",
+                    "properties": {
+                      "name": "[variables('applicationGatewayPublicIpName')]",
+                      "workspaceId": "[parameters('LogAnalyticsWorkspaceResourceID')]",
+                      "logs": [ 
+                        {
+                            "category": "DDoSProtectionNotifications",
+                            "enabled": true,
+                            "retentionPolicy": {
+                              "enabled": false,
+                              "days": 0
+                            }
+                          },  
+                          {
+                            "category": "DDoSMitigationFlowLogs",
+                            "enabled": true,
+                            "retentionPolicy": {
+                              "enabled": false,
+                              "days": 0
+                            }
+                          },{
+                            "category": "DDoSMitigationReports",
+                            "enabled": true,
+                            "retentionPolicy": {
+                              "enabled": false,
+                              "days": 0
+                            }
+                          }
+                      ],
+                      "metrics": [
+                        {
+                          "category": "AllMetrics",
+                          "enabled": true,
+                          "retentionPolicy": {
+                            "enabled": false,
+                            "days": 0
+                          }
+                        }
+                      ]
+                    }
+                  }
+            ]
+    },
+    {
+        "type": "Microsoft.Network/applicationGateways",
+        "name": "[variables('applicationGatewayName')]",
+        "apiVersion": "2018-08-01",
+        "location": "[resourceGroup().location]",
+        "properties": {
+            "sku": {
+                "name": "[parameters('skuName')]",
+                "tier":  "[parameters('skuTier')]",
+                "capacity": "[parameters('skuCapacity')]"
+            },
+            "enableHttp2":true,

```

Comment:
We probably need to call out that this won't allow us to do GRPC as it's not HTTP2 all the way through

Reply:
shall I add a comment to the json? :-)

Reply:
Maybe just something to track as an issue so everyone is aware when we come to GRPC stuff?

Reply:
added comment on #1027